.github/workflows/ci.yml

```
@@ -44,10 +44,14 @@ jobs:
 
       - uses: Swatinem/rust-cache@v1
 
-      - name: Run cargo test
+      - name: Install cargo-nextest
+        run: curl --location --silent --show-error --fail "https://github.com/alsuren/cargo-quickinstall/releases/download/cargo-nextest-0.9.14-x86_64-unknown-linux-gnu/cargo-nextest-0.9.14-x86_64-unknown-linux-gnu.tar.gz" | tar -xzvvf - -C /home/runner/.cargo/bin
```

```suggestion
      - name: Install cargo-nextest
        uses: taiki-e/install-action@nextest
```

ref: https://nexte.st/book/pre-built-binaries.html#using-nextest-in-github-actions